static/src/js/actions/project.js

```diff
@@ -338,131 +343,160 @@ export const getProjectCollections = () => async (dispatch, getState) => {
+      }
 
-        const accessMethodsObject = insertSavedAccessConfig(
-          buildAccessMethods(metadata, isOpenSearch),
-          savedAccessConfig
-        )
+      const focusedMetadata = createFocusedCollectionMetadata(
+        metadata,
+        authToken,
+        earthdataEnvironment
+      )
+
+      const isOpenSearch = !!getOpenSearchOsddLink(metadata)
+      payload.push({
+        abstract,
+        archiveAndDistributionInformation,
+        associatedDois,
+        boxes,
+        cloudHosted,
+        coordinateSystem,
+        dataQualitySummaries,
+        dataCenter,
+        duplicateCollections,
+        granules,
+        hasAllMetadata: true,
+        hasGranules,
+        id: conceptId,
+        isCSDA: isCSDACollection(dataCenters),
+        isOpenSearch,
+        relatedCollections,
+        services,
+        shortName,
+        subscriptions,
+        tags,
+        tilingIdentificationSystems,
+        title,
+        tools,
+        variables,
+        versionId,
+        ...focusedMetadata
+      })
 
-        const { methods = {} } = accessMethodsObject
-        let { selectedAccessMethod } = accessMethodsObject
+      const { [conceptId]: savedAccessConfig } = savedAccessConfigs
 
-        if (Object.keys(methods).length === 1 && !selectedAccessMethod) {
-          const [firstAccessMethod] = Object.keys(methods)
-          selectedAccessMethod = firstAccessMethod
-        }
+      const accessMethodsObject = insertSavedAccessConfig(
+        buildAccessMethods(metadata, isOpenSearch),
+        savedAccessConfig
+      )
+
+      const { methods = {} } = accessMethodsObject
+      let { selectedAccessMethod } = accessMethodsObject
+
+      if (Object.keys(methods).length === 1 && !selectedAccessMethod) {
+        const [firstAccessMethod] = Object.keys(methods)
+        selectedAccessMethod = firstAccessMethod
+      }
 
-        dispatch(actions.addAccessMethods({
-          collectionId: conceptId,
-          methods,
-          selectedAccessMethod
-        }))
+      dispatch(actions.addAccessMethods({
+        collectionId: conceptId,
+        methods,
+        selectedAccessMethod
+      }))
 
+      if (dataQualitySummaries) {
         const { items: dqsItems = [] } = dataQualitySummaries
         if (dqsItems) {
           dispatch(actions.setDataQualitySummaries({
             catalogItemId: conceptId,
             dataQualitySummaries: dqsItems
           }))
         }
-      })
+      }
 
       // Update metadata in the store
       dispatch(actions.updateCollectionMetadata(payload))
     })
-    .catch((error) => {
-      dispatch(actions.handleError({
-        error,
-        action: 'getProjectCollections',
-        resource: 'project collections'
-      }))
-    })
+  } catch (error) {
+    dispatch(actions.handleError({
+      error,
+      action: 'getProjectCollections',
+      resource: 'project collections'
+    }))
+  }
 
   return response
```

Comment:
Can this response be `null` (the initialized value?)

Reply:
That's what we do in line 346

Reply:
Yea that's what I'm wondering so if I'm reading correct, we are only going to return `null` if the try fails and we catch an error. If that is the case then I'd prefer we don't have a let but, instead like in `focusedCollection`
we just return `null` on line 501 makes it a easier to trace